Makefile

```diff
@@ -41,7 +41,11 @@ kurator: clean
 		cmd/kurator/main.go
 
 .PHONY: lint
-lint: golangci-lint lint-copyright
+lint: golangci-lint lint-copyright lint-markdown
+
+.PHONY: lint-markdown
+lint-markdown:
+	markdownlint docs --ignore docs/install-components -c mdl.json
```

Comment:
remove this `ignore` when we merged all content to new infra folder. 
cc @prodanlabs